src/nvim/ex_getln.c

```diff
@@ -1095,7 +1098,8 @@ static int command_line_wildchar_complete(CommandLineState *s)
 
     if (wim_flags[s->wim_index] & kOptWimFlagLongest) {
       res = nextwild(&s->xpc, WILD_LONGEST, options, s->firstc != '@');
-    } else if (wim_flags[s->wim_index] & kOptWimFlagFull) {
+    } else if ((wim_flags[s->wim_index] & kOptWimFlagFull)
+               && !(wim_flags[s->wim_index] & kOptWimFlagNoselect)) {
```

Comment:
?

Reply:
oh wait i think i see it now

Reply:
No it doesn't. The original change only changes the one in the other branch.